content/_data/changelogs/weekly.yml

```
@@ -12362,6 +12363,56 @@
   # pull: 5671 (PR title: Mark Resource Domain Recommendation monitor as security)
   # pull: 5676 (PR title: Bump mockito-core from 3.11.2 to 3.12.1)
 
+  - version: '2.309'
+    date: 2021-08-30
+    changes:
+      - type: TODO
+        category: developer
+        pull: 5682
+        authors:
+          - dependabot[bot]
+        pr_title: Bump jna from 5.8.0 to 5.9.0
+        message: |-
+          (No proposed changelog)
+      - type: rfe
+        category: internal
+        pull: 5687
+        authors:
+          - basil
+        pr_title: Mark `PluginFirstClassLoader`, `MaskingClassLoader`, and `AntClassLoader`
+          as parallel-capable
+        message: |-
+          Internal:
+          Internal: <code>AntClassLoader</code> (and its subclass <code>PluginFirstClassLoader</code>) and <code>MaskingClassLoader</code> register themselves as parallel-capable.
+      - type: rfe
+        category: rfe
+        pull: 5542
+        authors:
+          - MadsJakobsen
+        pr_title: Add `ABORTED` threshold to ReverseBuildTrigger
+        message: |-
+          Add <code>ABORTED</code> threshold to <code>ReverseBuildTrigger</code>
+      - type: bug
+        category: bug
+        pull: 5690
+        authors:
+          - janfaracik
+        pr_title: Fix missing icons
+        message: |-
+          Fix missing settings/cog icon in Plugin Manager
+           Fix incorrect folder icon showing in projects
+
+          Before
+          ![130775101-d4459d3b-f34a-4ebc-be94-caa60f5637f1](https://user-images.githubusercontent.com/43062514/130799962-9fbf59ae-10b2-40dd-92d6-7afcf00302a7.png)
+
+          After
+          <img width="324" alt="image" src="https://user-images.githubusercontent.com/43062514/130799920-0bfdabe5-4ed8-4844-9fa4-773e9df28514.png">
+          (regression in TODO)
```

Need to remove the images from the changelog.
```suggestion
      - type: bug
        category: bug
        pull: 5690
        authors:
          - janfaracik
        pr_title: Fix missing icons
        message: |-
          Fix missing settings/cog icon in Plugin Manager.
          Fix incorrect folder icon showing in projects (regression in 2.307).
```